### ISSUE TITLE
Implement Evaluation Metrics

### ISSUE THREAD

**Issue by jmcclary27** [jmcclary27/ACC-MLOps-Project#10]
Create:
- ml/evaluation/metrics.py

Use:
- from sklearn.metrics import classification_report

Log:
- Accuracy
- Micro F1
- Macro F1
- Per-class F1
- Print clean summary.

Return a plain dictionary with the metrics
